repository-initializer/initialize.sh

```diff
@@ -0,0 +1,2 @@
+apt-get update && apt-get install curl -y && apt-get clean
```

Comment:
I would remove this line and only keep the curl. Having curl seems very basic to me.